examples/basic_jit/main.cpp

```diff
@@ -42,7 +42,7 @@ static std::size_t estimateCodeSize(const zasm::Program& program)
         }
         else if (auto* nodeInstr = node->getIf<zasm::Instruction>(); nodeInstr != nullptr)
         {
-            size += nodeInstr->getLength();
+            size += 15;
```

Comment:
magic 15

Reply:
Good catch, I did that without commenting that its a temporary change.